build/tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
 // +build tools
```

Comment:
I don't understand these comments. They look almost the same. Did you want to replace
```
// +build tools
```
by
```
//go:build tools
```
?

Reply:
We cannot replace it as it won't properly for Go <= 1.16.
`go fmt` requires to have both lines.

Reply:
Okay. Good to know 🙂